test/e2e/redirects.fixture.ts

```diff
@@ -0,0 +1,291 @@
+const devSiteStaticChecksums = [
+  {
+    checksum: 'efcfc7a2314cbb5e9a8774bac786945e04fcd769',
+    uriPath: '/GeoIP2-Enterprise-CSV_Example.zip',
+  },
+  {
+    checksum: '77543b94ce06150b9c71149e50b07585d329d6f8',
+    uriPath: '/assets/svg/bg-pattern.svg',
+  },
+  {
+    checksum: '8754acf71336199677759db99ed63adf62a162d6',
+    uriPath: '/csv-files/GeoNames-Monthly-Diff-Report-April-2020.csv',
+  },
+  {
+    checksum: '08d55350215133f2ae6e4dd312af733a4fb2cb0f',
+    uriPath: '/csv-files/GeoNames-Monthly-Diff-Report-April-2021.csv',
+  },
+  {
+    checksum: '678f6f2d0bb1c1dfe9df1064005529a282c93495',
```

Comment:
Does this mean that whenever a csv diff report is added, this file needs to be updated?

Reply:
No, this is just for existing files that are being moved around as a part of this PR. After this runs in production, I think we can remove this.

Reply:
Actually, I think we could leave this as a way of making sure we don't remove historic files that are not actively referenced on the site but still need to exist because scripts or other sites might have links to them.